src/CIQS-CLI-Extension/azext_ciqs/custom.py

```diff
@@ -30,15 +54,42 @@ def deployDeployment(cmd, deploymentId, subscription=None):
     print('To be implemented')
 
 def viewDeployment(cmd, deploymentId, subscription=None):
```

Comment:
From the UX, it should look something like:
```
-----------------
Deployment Id: abcd-efghij-klmn-opqrstuv
Deployment Name: dlmt001
Subscription Id: swrw-wer2xc-234sd-wer23sdd
Location: West US
Status: ActionRequired
CurrentProvisioningStep: 003
InputParameters: {"ServerName" : xxx,"Database Name" : yyy}
-----------------
```
To achieve this, you might need to create a simplified version of "Deployment" class.